src/frontend/Komponenter/Behandling/Modal/HenleggModal.tsx

```diff
@@ -85,8 +121,20 @@ export const HenleggModal: FC<{ behandling: Behandling }> = ({ behandling }) =>
     const lukkModal = () => {
         settFeilmelding('');
         settVisHenleggModal(false);
+        settHenlagtårsak(undefined);
     };
 
+    const tilknyttetFullmakt = fullmakt.some(
+        (fullmakt) => fullmakt.gyldigTilOgMed === null || erEtterDagensDato(fullmakt.gyldigTilOgMed)
```

Comment:
Nå heter liste av fullmakter det samme som objektet vi itererer over. Dette kalles for shadowing og anbefales å unngå. Mitt forslag er at lista med flere fullmakter heter `fullmakter`